src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -88,6 +88,11 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+    	
+    	if(!path.toFile().exists()){
+    		throw new StorageOperationException("File is deleted!");
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.